.env.example

```diff
@@ -11,6 +11,7 @@ ENABLE_SCHOOLS_INTERFACE=false
 ENABLE_LOCAL_TLS=true
 TEST_GUIDANCE=true
 ENABLE_BLAZER=true
+ENABLE_API=true
```

Comment:
as discussed we'll probably need to move this to a feature flag to avoid deploying to enable/disable at the time of migration

Reply:
I think adding feature flags is a bigger change, happy for it to be done following this though.

Reply:
makes sense to be separate 👍